pkg/monitor/Dockerfile

```diff
@@ -0,0 +1,73 @@
+# syntax=docker/dockerfile:1
+ARG RUST_VERSION=lfedge/eve-rust:1.80.1
+FROM --platform=$BUILDPLATFORM ${RUST_VERSION} AS toolchain-base
+ARG TARGETARCH
+
+FROM toolchain-base AS target-amd64
+ENV CARGO_BUILD_TARGET="x86_64-unknown-linux-musl"
+
+FROM toolchain-base AS target-arm64
+ENV CARGO_BUILD_TARGET="aarch64-unknown-linux-musl"
+
+FROM toolchain-base AS target-riscv64
+ENV CARGO_BUILD_TARGET="riscv64gc-unknown-linux-gnu"
+
+FROM target-$TARGETARCH AS toolchain
+RUN echo "Cargo target: $CARGO_BUILD_TARGET"
+
+FROM toolchain AS planer
+# get sources from GIT
+ADD --keep-git-dir=true https://github.com/lf-edge/eve-monitor-rs.git /app
+
+WORKDIR /app
+# create a recipe
+RUN cargo chef prepare --recipe-path recipe.json
+
+FROM toolchain AS cacher
+# copy the recipe
+WORKDIR /app
+COPY --from=planer /app/recipe.json recipe.json
+# build the dependencies
+RUN cargo chef cook --release --recipe-path recipe.json
+
+# building the final image
+FROM toolchain AS builder
+ADD --keep-git-dir=true https://github.com/lf-edge/eve-monitor-rs.git /app
+WORKDIR /app
+
+# copy prebuilt dependencies
+# and the cargo directory with crates.io index
+COPY --from=cacher /app/target /app/target
+COPY --from=cacher $CARGO_HOME $CARGO_HOME
+RUN echo "Cargo target: $CARGO_BUILD_TARGET"
+
+RUN cargo build --release
+RUN cargo sbom > sbom.spdx.json
+RUN cp /app/target/$CARGO_BUILD_TARGET/release/monitor /app/target/
+
+
+FROM lfedge/eve-alpine:1f7685f95a475c6bbe682f0b976f12180b6c8726 AS runtime
```

Comment:
We use 591df01e581889c3027514c8a91feaca1c8ad49f now.

Reply:
@OhmSpectator yeah, CI told me that. I'll fix.